services/web/server/src/simcore_service_webserver/projects/projects_settings.py

```diff
@@ -12,3 +16,8 @@ class ProjectsSettings(BaseCustomSettings):
         default=5,
         description="defines the number of dynamic services in a project that can be started concurrently (a value of 0 will disable it)",
     )
+
+    PROJECTS_DYNAMIC_SERVICES_REDIS_LOCK_TIMEOUT_S: NonNegativeInt = Field(
```

Comment:
Ok so here first I think the description is incorrect.
If I get that right this is:
- a Max waiting time to acquire the distributed lock correct?
- if that is correct this does not wait for the service to start only to trigger the starting process correct?
Suggestions:
- use a datetime.timedelta, then we can get rid of S at the end and the ops can put either seconds or ```00:10:00``` or a ISO timestamp
- on line 15 we have ```PROJECTS_MAX_NUM_RUNNING_DYNAMIC_NODES``` since these are nodes, why use right after that services?
- I see the problem to name this... I'm also wondering if this is a value that needs to be changed in the settings at all.
- Can you measure the time to make this call vs the max number of services? if yes then you should be able to compute that number on the fly. and there is no need to change it. How am I to find that number as a devOps?